src/pages/developing/frameworks/react.mdx

```diff
@@ -6,6 +6,12 @@ tabs: ['Web Components', 'React']
 
 import { Link } from 'gatsby';
 
+<InlineNotification>
```

Comment:
```suggestion
<InlineNotification kind="warning">
```

@annawen1 Do we want to try a different notification type?

@oliviaflory @RichKummer unless you guys wanna stick with this default `info` one.

Reply:
hmmm I think warning is good!